Add a newline at the end of NiHandTracker/main.cpp.

From: Roman Donchenko <roman.donchenko@itseez.com>

This patch is originally from <https://kforge.ros.org/openni/drivers/>.
---
 Samples/NiHandTracker/main.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Samples/NiHandTracker/main.cpp b/Samples/NiHandTracker/main.cpp
index dad16f3..4c79104 100644
--- a/Samples/NiHandTracker/main.cpp
+++ b/Samples/NiHandTracker/main.cpp
@@ -75,4 +75,4 @@ int main(int argc, char* argv[])
 	}
 
 	return 0;
-}
\ No newline at end of file
+}
